com.unity.render-pipelines.universal/Runtime/UniversalRenderer.cs

```diff
@@ -123,8 +152,29 @@ public UniversalRenderer(UniversalRendererData data) : base(data)
             m_ForwardLights = new ForwardLights();
             //m_DeferredLights.LightCulling = data.lightCulling;
             this.m_RenderingMode = data.renderingMode;
+            this.m_DepthPrepassMode = data.depthPrepassMode;
+            this.m_DepthPrimingMode = data.depthPrimingMode;
             this.usesRenderPass = data.useNativeRenderPass;
 
+#if !UNITY_ANDROID && !UNITY_IOS
+            this.m_CanUseDepthPriming = true;
+
+#if UNITY_EDITOR_WIN || UNITY_STANDALONE_WIN
+            if (CultureInfo.InvariantCulture.CompareInfo.IndexOf(SystemInfo.graphicsDeviceVendor, "ARM", CompareOptions.IgnoreCase) >= 0)
+            {
+                this.m_CanUseDepthPriming = false;
+            }
+#elif UNITY_EDITOR_OSX || UNITY_STANDALONE_OSX
+            if (CultureInfo.InvariantCulture.CompareInfo.IndexOf(SystemInfo.graphicsDeviceVendor, "SILICON", CompareOptions.IgnoreCase) >= 0)
+            {
+                this.m_CanUseDepthPriming = false;
+            }
```

Comment:
Why is it disabled in OSX?

Reply:
This was a crude and ugly way to check if a system might have TBDR hardware.

I've replaced this whole section with `m_CanUseDepthPriming = !SystemInfo.hasHiddenSurfaceRemovalOnGPU;`
This still leaves us with Depth Priming on Tiled GPU's without HSR. To detect those we should expose an internal property.
The assumption is that for Tiled GPUs:

- If HSR is also supported then Depth Priming will have no benefits.
- If some form of automated sorting is present we can potentially still gain a small benefit from Depth Priming.
- If MSAA is in play then we would have to perform prohibitively expensive MSAA depth load/stores unless we can merge the depth and opaque passes together.

For now I would proceed as follows:
- Disable Depth Priming on TBDR hardware. (current state of the PR)
- Expose the required info in Unity to allow us to detect Tile GPUs and then disable it for those as well. (so in a later PR)
- Add in a special case for Tiled Hardware without HSR: allow if no MSAA or if the Depth and Opaque passes can be merged. (another later PR)

Reply:
Makes sense to me.